save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -74,6 +80,10 @@ class DemoManagerController(
         .switchIfEmptyToNotFound {
             "Could not find project $projectName in organization $organizationName."
         }
+        .filter { projectPermissionEvaluator.hasPermission(authentication, it, Permission.DELETE) }
```

Comment:
we have the method: `requireOrSwitchToResponseException`. you can try to use it here.
```suggestion
        .requireOrSwitchToResponseException({ projectPermissionEvaluator.hasPermission(authentication, it, Permission.DELETE) }, HttpStatus.FORBIDDEN) {
            "Not enough permission for accessing given project."
        }
```

Reply:
Done [4a36dc6](https://github.com/saveourtool/save-cloud/pull/1688/commits/4a36dc64bb2f0c58be27a075e672c98364c1daac)